File: FrequencyCounter-0(N^2).js
let list1 = [1,2,3,2]
let list2 = [9,1,4,4]

function same (list1,list2){
    if (list1.length != list2.length){
        return false
    }
    for (let i=0; i==list1.length; i++){
        let correctIndex = list2.indexOf(list1[i]**2)
        if (correctIndex === -1){
            return false
        }
        list2.splice(correctIndex,1)
    }
    return true
}

File: FrequencyCounter/FrequencyCounter-0(N^2).py
list1 = [1,2,3,2]
list2 = [9,1,4,4]

def same(list1, list2):
    if len(list1) != len(list2):
        return False
    
    for i in list1:
        squaredNumber = i ** 2
        if squaredNumber not in list2:
            return False
        else:
            list2.remove(squaredNumber)
    return True

a = same(list1, list2)
print(a)
